pkg/ocm/output/nodepools.go

```diff
@@ -33,6 +33,15 @@ func PrintNodePoolAutoscaling(autoscaling *cmv1.NodePoolAutoscaling) string {
 	return output.No
 }
 
+func PrintNodePoolReplicasInline(autoscaling *cmv1.NodePoolAutoscaling, replicas int) string {
```

Comment:
@chenz4027 If you could add a quick unit test for this one, that would be awesome. Thanks!

Reply:
Sure @robpblake , will do 👍